csharp/src/Ice/Util.cs

```diff
@@ -277,7 +277,11 @@ public static Logger getProcessLogger()
     {
         lock (_processLoggerMutex)
         {
-            _processLogger ??= new ConsoleLoggerI(AppDomain.CurrentDomain.FriendlyName);
+            if (_processLogger is null)
+            {
+                _processLogger = new ConsoleLoggerI(AppDomain.CurrentDomain.FriendlyName);
```

Comment:
If setProcessLogger is not called this is never disposed, I guess there isn't a good place to dispose of statics like this.

Reply:
Yes, static fields are not disposed.